LICENSE

```diff
@@ -210,6 +210,12 @@ This product bundles some test examples from the Stix project (metron-platform/m
 
 This product bundles wait-for-it.sh, which is available under a "MIT Software License" license.  For details, see https://github.com/vishnubob/wait-for-it
 
+------------------------------------------------------------------------------------
+
```

Comment:
Updated with another license.  The testing DB used is under a share-alike 3.0 (not 4.0) license, so I explicitly called it it and added a link to the GitHub repo.